kouncil-frontend/package.json

```diff
@@ -26,6 +26,7 @@
     "@nrwl/angular": "15.2.1",
     "@stomp/rx-stomp": "1.1.4",
     "@swimlane/ngx-datatable": "20.1.0",
+    "avro-typescript": "^1.1.0",
```

Comment:
Last version of this lib was published 4 year ago. Are you sure we want to use it?

Reply:
There is not so many good libs for avro. That one has the most functionalities we need. Maybe at some point later it would be good to prepare internal lib for that